src/components/PivotTable/PivotTableValueCell.js

```diff
@@ -58,13 +58,18 @@ export const PivotTableValueCell = ({
         maxWidth: width,
     }
 
+    const classes = [cellContent.cellType, cellContent.valueType]
```

Comment:
Logic looks fine to me, might be a little cleaner like this:

```js
const isClickable = cellContent.cellType === CELL_TYPE_VALUE && cellContent.ouId
const onClick = isClickable ? () => { onToggleContextualMenu(cellRef, cellContent) } : undefined
const classes = [cellContent.cellType, cellContent.valueType, isClickable && 'clickable']

//...

return <PivotTableCell
            key={column}
            classes={classes}
            title={cellContent.renderedValue}
            style={style}
            onClick={onClick}
            ref={cellRef}
/>
```

Reply:
Cleaned up.